tauri/src/app.rs

```diff
@@ -92,10 +95,11 @@ impl IronApp {
     }
 
     pub fn run(&mut self) {
-        self.app.take().unwrap().run(|_, event| {
-            if let tauri::RunEvent::ExitRequested { api, .. } = event {
+        self.app.take().unwrap().run(|_, event| match event {
+            tauri::RunEvent::ExitRequested { api, .. } => {
                 api.prevent_exit();
             }
+            _ => {}
```

Comment:
why this change? as you can see, clippy is already asking to change it back

Reply:
oh, sorry, I was going to handle the event here